angular-ngrx-scss/src/app/user/services/user.service.ts

```diff
@@ -77,7 +77,7 @@ export class UserService {
   getUserRepos(username: string): Observable<UserReposState[]> {
     const url = `${environment.githubUrl}/users/${encodeURIComponent(
       username,
-    )}/repos`;
+    )}/repos?sort='created'`;
 
     return this.http.get<UserReposApiResponse>(url).pipe(
```

Comment:
Can we pass query params with the `this.http.get` call instead?

Reply:
Should be possible! Going to move this into a separate ticket so the Amplify fix can be merged in.